api/Dockerfile-Serdes

```diff
@@ -0,0 +1,38 @@
+ARG BASE_UI_IMAGE=ghcr.io/kafbat/kafka-ui:main
```

Comment:
This means that if we run this workflow without running our main image, we're gonna build an outdated image. We need to either depend on the base image workflow or use some other approach.

Reply:
it is possible, but I think it is better approach
this enables to track versioning of serders separatly from the kafka-ui main.